packages/core/parcel-bundler/src/Server.js

```diff
@@ -57,9 +57,7 @@ function middleware(bundler) {
         !pathname.startsWith(bundler.options.publicURL) ||
         path.extname(pathname) === ''
       ) {
-        // If the URL doesn't start with the public path, or the URL doesn't
-        // have a file extension, send the main HTML bundle.
-        return sendIndex();
+        sendIndex();
```

Comment:
why did you drop the `return` here? It looks like it doesn’t matter, but seems like the code should be consistent with the other if-statement branches here.

```suggestion
        return sendIndex();
```

Reply:
Wasn't done on purpose, must've not noticed the inconsistency. I've applied the suggestion